File: pgm/ABC_Visual_Basic/ABC_155_A.vb
Public Class Main
    Public Shared Sub Main(ByVal args() As String)
        Dim A, B, C As Integer
        Dim arr(3) As String
        arr = Split(Console.ReadLine())
        A = arr(0) : B = arr(1) : C = arr(2)
        Dim flg As Boolean = False
        If A = B AndAlso B <> C Then
            flg = True
        End If
        If B = C AndAlso C <> A Then
            flg = True
        End If
        If C = A AndAlso A <> B Then
            flg = True
        End If
        Console.WriteLine(If(flg, "Yes", "No"))
    End Sub
End Class

File: pgm/ABC_C#/ABC_155_A.cs
using System;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        int A = int.Parse(arr[0]), B = int.Parse(arr[1]), C = int.Parse(arr[2]);
        bool flg = false;
        if (A == B && B != C)
            flg = true;
        else if (B == C && C != A)
            flg = true;
        else if (C == A && A != B)
            flg = true;
        Console.WriteLine(flg ? "Yes" : "No");
    }
}